r-package/grf/tests/testthat/data/README.md

```diff
@@ -0,0 +1,36 @@
+# Datasets
+
+The datasets is this directory were generated through the following R scripts:
```

Comment:
I'm sorry about my incorrect suggestion earlier -- I read too fast and didn't see that all of these CSV files are simply used in characterization tests. This should have been obvious since they all end in `_predictions`!

We could omit the README and go with what you had before then, since it nicely matches the C++ characterization tests.

Reply:
Oki, I've deleted this file and added the details to the R test file instead